gst/nnstreamer/nnstreamer_subplugin.h

```diff
@@ -46,17 +46,6 @@ typedef enum {
 
 #define NNS_SUBPLUGIN_CHECKER (0xdeadbeef)
 
-/**
- * @brief Any independent shared object subplugin must export this as
- * "nnstreamer_subplugin"
- */
-typedef struct {
-  uint32_t checker; /**< This MUST be 0xdeadbeef (NNS_SUBPLUGIN_CHECKER) */
-  subpluginType type; /**< The subplugin type */
-  const char *name; /**< The name of subplugin. The resulting .so file should be ${prefix}${name}.so */
-  void *data; /**< Subplugin specific data (usually funcptr callbacks), forwarded to subplugin's corresponding handler */
-} nnstreamer_subplugin_data;
```

Comment:
Please update line 28 later (nnstreamer_subplugin_data struct is removed)

Reply:
Updated!